soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -0,0 +1,1944 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CHIP_CHIPREGS_H
+#define CHIP_CHIPREGS_H
+
+#include <sys/util.h>
+
+#define EC_Register_Base_Address DT_REG_ADDR_BY_IDX(DT_NODELABEL(baseaddr), 0)
```

Comment:
would prefer if this was all caps i.e. EC_REGISTER_BASE_ADDRESS (or even EC_REG_BASE_ADDR)